cime/src/drivers/mct/main/prep_glc_mod.F90

```diff
@@ -342,6 +458,21 @@ subroutine prep_glc_accum_avg(timer)
        end do
     end if
     l2gacc_lx_cnt = 0
+
+    do egi = 1,num_inst_glc
+       ! temporary formation of average
+       if (x2gacc_gx_cnt > 1) then
+	  call mct_avect_avg(x2gacc_gx(egi), x2gacc_gx_cnt)
+       end if
+
+       ! ***NOTE***THE FOLLOWING ACTUALLY MODIFIES x2g_gx
```

Comment:
@JeremyFyke , this comment doesn't sound good.  Do we need to investigate a different implementation here?

Reply:
Can you grep for this comment string elsewhere?  I *suspect* it is not unique to this occurrence.  If not, I'll investigate further.   While it would be good to do a 'proper' implementation, I would ask @rljacob to check this.  It may not be STRICTLY necessary, as I recall (otherwise I would have done it, in theory)

Reply:
What is the glacier model coupling frequency?  Nearly all the other components, including ocean, are coupling every hour now.

Reply:
@rljacob, right now we have GLC coupling once per day.  It could be longer (e.g. a week or two), but a day is easier to deal with for now.  The disadvantage to making it shorter is that the ice sheet time step is order weeks, so a really small coupling interval would require running the ice sheet model much, much more frequency than necessary, at associated expense.